packages/forma-36-react-components/src/components/Card/Card.tsx

```diff
@@ -46,57 +46,55 @@ interface CardPropTypes extends BaseCardProps {
    * Any additional styles that are being applied
    */
   style?: React.CSSProperties;
-  children: React.ReactNode;
 }
 
-const defaultProps: Partial<CardPropTypes> = {
+const defaultProps: Partial<CardProps> = {
   padding: 'default',
   testId: 'cf-ui-card',
   selected: false,
 };
 
-export class Card extends Component<CardPropTypes> {
-  static defaultProps = defaultProps;
+export const Card = ({
+  className,
+  testId,
+  children,
+  href,
+  target,
+  onClick,
+  padding,
+  selected,
+  ...otherProps
+}: CardProps) => {
+  const classNames = cn(styles.Card, className, {
+    [styles[`Card--padding-${padding}`]]: padding,
+    [styles['Card--is-interactive']]: onClick || href,
+    [styles['Card--is-selected']]: selected,
+  });
 
-  handleClick = (event: ReactMouseEvent<HTMLElement>) => {
-    if (!this.props.onClick) return;
+  const Element: React.ElementType = href ? 'a' : 'div';
 
-    this.props.onClick(event);
-  };
-
-  render() {
-    const {
-      className,
-      testId,
-      children,
+  return React.createElement(
+    Element,
+    {
       href,
-      target,
-      onClick,
-      padding,
-      selected,
-      ...otherProps
-    } = this.props;
-
-    const classNames = cn(styles.Card, className, {
-      [styles[`Card--padding-${padding}`]]: padding,
-      [styles['Card--is-interactive']]: onClick || href,
-      [styles['Card--is-selected']]: selected,
-    });
-
-    const Element: React.ReactType = href ? 'a' : 'div';
-    return React.createElement(
-      Element,
-      {
-        href,
-        target: href && target,
-        className: classNames,
-        onClick: this.handleClick,
-        'data-test-id': testId,
-        ...otherProps,
-      },
-      children,
-    );
-  }
-}
+      target: href && target,
+      'data-test-id': testId,
+      className: classNames,
+      ...(!!onClick && {
+        tabindex: '0', // necessary to make the div accessible with tab navigation
+        onClick: (event: MouseEvent<HTMLElement>) => onClick(event),
+        onKeyDown: (event: KeyboardEvent<HTMLElement>) => {
+          if (event.keyCode === 13) {
```

Comment:
[`event.keyCode` is deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode) and key code numbers are hard to understand. What about using [event.code](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code) instead? `event.code === "Enter"`

Reply:
Also, something tells me usual a11y practices would say the card should be selectable with the space key.

Reply:
Oh I didn't know it was going to be deprecated!

I will change that and implement the space key as well
thanks 🙌